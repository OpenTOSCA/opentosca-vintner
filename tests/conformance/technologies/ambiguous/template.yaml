tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            type_pruning: true
            ambiguous_technology_check: true

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      conditions: true
                - terraform:
                      conditions: true
