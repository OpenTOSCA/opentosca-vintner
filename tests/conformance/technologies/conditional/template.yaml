tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            optimization_technologies: true

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      conditions: true
                - terraform:
                      conditions: false
