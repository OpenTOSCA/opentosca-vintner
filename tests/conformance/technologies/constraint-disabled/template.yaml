tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            required_technology_constraint: false

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      conditions: true
                - terraform:
                      conditions: true
