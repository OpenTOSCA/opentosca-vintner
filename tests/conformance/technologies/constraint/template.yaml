tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            required_technology_constraint: true

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      conditions: true
                - terraform:
                      conditions: true
