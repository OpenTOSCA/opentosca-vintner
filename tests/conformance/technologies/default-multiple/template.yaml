tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            type_pruning: true

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      default_alternative: true
                - terraform:
                      default_alternative: true
