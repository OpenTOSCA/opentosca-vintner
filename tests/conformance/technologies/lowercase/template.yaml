tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            optimization_technologies: true

    node_templates:
        container:
            type: container
            technology:
                - Ansible:
                      conditions: true
