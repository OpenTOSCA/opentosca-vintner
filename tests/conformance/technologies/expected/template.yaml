tosca_definitions_version: tosca_variability_1_0

topology_template:
    variability:
        options:
            optimization_technologies: false
            technology_pruning: false
            technology_constraint: false

            expected_technology_check: true

    node_templates:
        container:
            type: container
            technology:
                - ansible:
                      conditions: false
